.github/workflows/process-load-test-result.yml

```diff
@@ -51,11 +54,11 @@ jobs:
       - name: Merge PR
         if: ${{ steps.processResults.outputs.hasPassed }}
         run: |
-          status="NOT COMPLETED"
-          while [ ! -z "$status" ]
+          checkCount="0"
+          while [ "$checkCount" != "3" ]
```

Comment:
@dilanSachi I hope, here the 3 is referred to the number of required checks?.

Reply:
yes

Reply:
Wouldn’t this loop executive forever if some checks have failed?.

Reply:
Added a check for that https://github.com/ballerina-platform/module-ballerina-grpc/pull/712/commits/9ace56f75b4d94f75e0d529d789da56e51791d1f